snap/info.go

```diff
@@ -165,6 +166,9 @@ type Info struct {
 	Plugs            map[string]*PlugInfo
 	Slots            map[string]*SlotInfo
 
+	// Plugs or slots with issues (they are not included in Plugs or Slots)
+	BadInterfaces map[string]string // slot or plug => message
```

Comment:
Shall we store the real error objects here? This way we could generate i18n specific message later.

Reply:
I'm not thrilled about storing error objects here, because these problems with interfaces are not actually treated as errors. Also, nothing is stopping us from populating this map with i18n, is it?

Reply:
The errors can be stored and transformed to per-request i18n message later. We cannot translate them at this time.

Reply:
We probably don't need to store those errors. Instead, if the interface definition is completely broken, we can reject the snap loading with the actual error.

Reply:
@niemeyer We have two possible problems to deal with: an unknown interface or interface not passing sanitization. AFAIR we didn't want to reject snaps on unknown interfaces. 
Shall we only reject snaps that fail sanitization?

Reply:
It's okay, let's keep it this way for now. Strings are fine for the time being (it is already like that in master). Easy to change when we want to.